firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -282,7 +282,7 @@ public static FirebaseCrashlytics getInstance() {
    */
   public void recordException(@NonNull Throwable throwable) {
     if (throwable == null) { // Users could call this with null despite the annotation.
-      Logger.getLogger().w("Crashlytics is ignoring a request to log a null exception.");
+      Logger.getLogger().v("A null value was passed to recordException. Ignoring.");
```

Comment:
This feels more like DEBUG (or WARN) than VERBOSE, since this indicates something unexpected.

Reply:
👍  after thinking about this a bit more I think you're right.